### PR TITLE
Change payment method name from "Wallet" to "Bitcoin"

### PR DESCRIPTION
**Pull request by phershbe**
As discussed here: https://github.com/btcpayserver/btcpayserver/issues/3571

### CODE REVIEW

**Review by pavlenex — APPROVED** [review]
Looks good to me! 🎉 

By the way you can test your own PR's by [building them locally](https://docs.btcpayserver.org/Development/LocalDevelopment/).
Screenshot 2022-03-25 at 09 03 19

File: BTCPayServer/Components/MainNav/Default.cshtml [hunk 1]
```diff
@@ -60,14 +60,14 @@
                                         {
                                             <a asp-area="" asp-controller="UIWallets" asp-action="WalletTransactions" asp-route-walletId="@scheme.WalletId" class="nav-link js-scroll-trigger @ViewData.IsActiveCategory(typeof(WalletsNavPages), scheme.WalletId.ToString()) @ViewData.IsActivePage(StoreNavPages.OnchainSettings)" id="@($"StoreNav-Wallet{scheme.Crypto}")">
                                                 <span class="me-2 btcpay-status btcpay-status--@(scheme.Enabled ? "enabled" : "pending")"></span>
-                                                <span>@(Model.AltcoinsBuild ? $"{scheme.Crypto} " : "")Wallet</span>
+                                                <span>@(Model.AltcoinsBuild ? $"{scheme.Crypto} " : "")Bitcoin</span>
```
Comment (dennisreimann) [inline BTCPayServer/Components/MainNav/Default.cshtml:63]:
```suggestion
                                                <span>@(Model.AltcoinsBuild ? $"{scheme.Crypto} Wallet" : "Bitcoin")</span>
```

File: BTCPayServer/Components/MainNav/Default.cshtml [hunk 2]
```diff
@@ -60,14 +60,14 @@
...
                                             </a>
                                         }
                                         else
                                         {
                                             <a asp-area="" asp-controller="UIStores" asp-action="SetupWallet" asp-route-cryptoCode="@scheme.Crypto" asp-route-storeId="@Model.Store.Id" class="nav-link @ViewData.IsActivePage(StoreNavPages.OnchainSettings)" id="@($"StoreNav-Wallet{scheme.Crypto}")">
                                                 <span class="me-2 btcpay-status btcpay-status--@(scheme.Enabled ? "enabled" : "pending")"></span>
-                                                <span>@(Model.AltcoinsBuild ? $"{scheme.Crypto} " : "")Wallet</span>
+                                                <span>@(Model.AltcoinsBuild ? $"{scheme.Crypto} " : "")Bitcoin</span>
```
Comment (dennisreimann) [inline BTCPayServer/Components/MainNav/Default.cshtml:70]:
```suggestion
                                                <span>@(Model.AltcoinsBuild ? $"{scheme.Crypto} Wallet" : "Bitcoin")</span>
```

**Review by dennisreimann — CHANGES_REQUESTED** [review]
We need to adapt it like this, because for the altcoin build we want to show e.g. "LTC Wallet".

**Review by Kukks — APPROVED** [review]

**Comment by pavlenex** [comment]
Congrats on your first contribution @phershbe here's to many more 🍻 🎉 🚀

**Comment by phershbe** [comment]
@pavlenex Thank you for the information about building them locally, I know Django and Node and how to build them locally but not .NET. I'll go through the link that you provided!

File: BTCPayServer/Components/MainNav/Default.cshtml (hunk 1 above)
Comment (phershbe) [reply BTCPayServer/Components/MainNav/Default.cshtml]: @dennisreimann Okay sorry about that. I understand now that it checks for altcoins and then renders either the name of the altcoin or "Bitcoin" depending on the result.

**Pull request merged**